wallet/src/Ergvein/Wallet/Page/History.hs

```diff
@@ -240,7 +240,7 @@ transactionsGetting cur = do
   pubSD <- getPubStorageD
   let allBtcAddrsD = ffor pubSD $ \(PubStorage _ cm _ _) -> case Map.lookup BTC cm of
         Nothing -> []
-        Just (CurrencyPubStorage keystore txmap _ _ _ _) -> extractAddrs keystore
+        Just (CurrencyPubStorage keystore txmap _ _ _ _ _) -> extractAddrs keystore
```

Comment:
Improve here: `CurrencyPubStorage keystore txmap _ _ _ _ _` to `CurrencyPubStorage{..}`